crates/ruff_dev/src/format_dev.rs

```diff
@@ -239,78 +261,88 @@ fn format_dev_multi_project(args: &Args) -> anyhow::Result<bool> {
         None => None,
     };
 
-    #[allow(clippy::print_stdout)]
     for project_path in project_paths {
-        bar.suspend(|| println!("Starting {}", project_path.display()));
-
-        match format_dev_project(
-            &[project_path.clone()],
-            args.stability_check,
-            args.write,
-            false,
-        ) {
+        info!(parent: None, "Starting {}", project_path.display());
+
+        match format_dev_project(&[project_path.clone()], args.stability_check, args.write) {
             Ok(result) => {
                 total_errors += result.error_count();
                 total_files += result.file_count;
 
-                bar.suspend(|| {
-                    println!(
-                        "Finished {}: {} files, similarity index {:.3}, {:.2}s",
-                        project_path.display(),
-                        result.file_count,
-                        result.statistics.similarity_index(),
-                        result.duration.as_secs_f32(),
+                info!(
+                    parent: None,
+                    file_count = result.file_count,
+                    similarity_index = result.statistics.similarity_index(),
+                    duration = result.duration.as_secs_f32(),
```

Comment:
Why are we passing all these arguments here? Do we want to filter by these?

Reply:
i was experimenting with progress bars here